### DISCUSSION TITLE
Next POI announcement

### DISCUSSION THREAD

**Discussion by airald** [devemux86/cruiser#617] [Q&A] [answered]
I've got a question about next close POIs. When you tap over the symbol, the name appears on the screen. And If I'm not wrong, the same text was spoken through TTS. What do now (not just after version 5, I saw 2 weeks ago using the last v4 version) is tell "gas station". For me is usefull to tell the name because I can stop on my prefferred petrol station. Should be possible to read the name instead the POI type?

**Comment by devemux86** [comment]
@airald 

I think we discussed and decided some time ago to speak the POI type,
as the name is not certain to be spoken correctly in the voice language?

e.g. English voice language speaks German names

We display the POI name (if it is available).

**Reply by airald** [reply]
Ok, so I was wrong, no problem.

**Reply by devemux86** [reply]
There is no problem to speak both the type and the name (if exists).

We just need to decide what we want. :slightly_smiling_face:

**Comment by devemux86** [comment]
We can try in the next update to speak Type + Name for fuel / charging stations and see how it sounds.

**Reply by DolomitesTom** [reply]
but only optional :-) ... by the way: how can I deactivate the speaking of "calculate new route"?

**Reply by devemux86** [reply]
This is about the POI announcement when tapping the POI panel,
so it is a manual and optional action.

The automatic announcement of nearby POI (and favorites)
are controlled in "Settings | Voice guidance | Announcement".

> how can I deactivate the speaking of "calculate new route"?

"Settings | Voice guidance | Advanced | Off route warning".

**Comment by devemux86** [comment, marked as answer]
- Available in https://github.com/devemux86/cruiser/discussions/641
